pyproject.toml

```
@@ -1,65 +1,63 @@
-[tool.poetry]
-name = "navigator_backend"
-version = "1.26.0"
+[project]
+name = "navigator-backend"
```

<!-- Questions are used to seek clarification or more information about the code.  -->
**question:** what's the reason for this change?

---

None whatsoever - just a mistake when converting it
```suggestion
name = "navigator_backend"
```